docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,734 @@
+
+In a similar manner, the :class:`~qiskit.opflow.MatrixEvolution` class performs evolution by classical matrix exponentiation,
+constructing a circuit with :class:`~UnitaryGate`\s or :class:`~HamiltonianGate`\s containing the exponentiation of the operator.
+This class is no longer necessary, as the :class:`~HamiltonianGate`\s can be directly handled by the algorithms.
+
+.. list-table:: Migration of ``qiskit.opflow.evolutions (2/2)``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import MatrixEvolution, MatrixOp
+
+            hamiltonian = MatrixOp([[0, 1], [1, 0]])
+            evolution = MatrixEvolution()
+            evol_result = evolution.convert(hamiltonian.exp_i())
+            evolved_state = evol_result.to_circuit()
+     -
+
+        .. code-block:: python
+
+            from qiskit.quantum_info import SparsePauliOp
+            from qiskit.extensions import HamiltonianGate
+            from qiskit import QuantumCircuit
+
+            evol_gate = HamiltonianGate([[0, 1], [1, 0]], 1)
+            evolved_state = QuantumCircuit(1)
+            evolved_state.append(evol_gate, [0])
+
+To summarize:
+
+.. list-table:: Migration of ``qiskit.opflow.evolutions.trotterizations``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.TrotterizationFactory`
+     - This class is no longer necessary.
+     -
+   * - :class:`~qiskit.opflow.Trotter`
+     - :class:`~synthesis.SuzukiTrotter` or :class:`~synthesis.LieTrotter`
+     -
+   * - :class:`~qiskit.opflow.Suzuki`
+     - `:class:`~synthesis.SuzukiTrotter`
+     -
+   * - :class:`~qiskit.opflow.QDrift`
+     - :class:`~synthesis.QDrift`
+     -
+
+.. list-table:: Migration of ``qiskit.opflow.evolutions.evolutions``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.EvolutionFactory`
+     - This class is no longer necessary.
+     -
+   * - :class:`~qiskit.opflow.EvolvedOp`
+     - :class:`~synthesis.SuzukiTrotter`
+     - This class is no longer necessary
+   * - :class:`~qiskit.opflow.MatrixEvolution`
+     - :class:`~HamiltonianGate`
+     -
+   * - :class:`~qiskit.opflow.PauliTrotterEvolution`
+     - :class:`~PauliEvolutionGate`
+     -
+
+Expectations
+------------
+Expectations are converters which enable the computation of the expectation value of an observable with respect to some state function.
+This functionality can now be found in the estimator primitive.
+
+Algorithm-Agnostic Expectations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table:: Migration of ``qiskit.opflow.expectations``
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+   * - :class:`~qiskit.opflow.ExpectationFactory`
+     - No direct replacement.
+     -
+   * - :class:`~qiskit.opflow.AerPauliExpectation`
+     - Use :class:`~Estimator` primitive from ``qiskit_aer`` with ``approximation=True``
```

Comment:
And `shots=None` - the AerPauliExpectation gave an exact answer using Aer's special expectation instruction I think it needs both shots=None and approximation=True for the same net outcome (though its confusing to me that having something called approximation True gives an Exact outcome!!)